apps/annotation/opencv_annotation.cpp

```diff
@@ -278,15 +286,31 @@ int main( int argc, const char** argv )
     for (size_t i = 0; i < filenames.size(); i++){
         // Read in an image
         Mat current_image = imread(filenames[i]);
+        bool resize_bool = false;
```

Comment:
I think it's worth to make this const too.

    bool const resize_bool = maxWindowHeight > 0) && (current_image.rows > maxWindowHeight);

or maybe extract a function for the conditional if you want to avoid repeating it..